### PR TITLE
Better blur

### PR DESCRIPTION
**Pull request by bradparker**
Higher order component for aggregating focus / blur events coming from a component with many focusable children.

### CODE REVIEW

**Comment by slwen** [comment]
Looks good :thumbsup: Do you have an example of how this works with FilterSelect?

**Comment by bradparker** [comment]
Hmmm, sure I'll make a wee gif

**Comment by bradparker** [comment]
focus-and-blur

Basically you can blur the FilterSelect / OptionList and anything which uses them.

File: forms/AddressLookup/index.js [hunk 1]
```diff
@@ -126,6 +126,12 @@ export default React.createClass({
     })
   },
 
+  handleCountrySelectBlur() {
+    this.setState({
```
Comment (alexander-bobin) [inline forms/AddressLookup/index.js]: Is this worth unit testing?  Doesn't look like we have many tests for this file anyways

Comment (bradparker) [inline forms/AddressLookup/index.js]: Hrm, added a spec... I need a way to mock components which use the aggregate focus component for specs. I don't want to know about the delay in tests. But will have to have a think about it.

**Pull request merged**